src/haddock/modules/analysis/caprieval/__init__.py

```diff
@@ -23,6 +27,31 @@ def confirm_installation(cls):
         """Confirm if contact executable is compiled."""
         return
 
+    def _rearrange_output(self, output_name, path, ncores):
+        """Combine different capri outputs in a single file."""
+        output_fname = Path(path, output_name)
+        self.log(f"rearranging output files into {output_fname}")
+        keyword = output_name.split(".")[0]
+        split_dict = {
+            "capri_ss": "model-cluster-ranking",
+            "capri_clt": "caprieval_rank"
+            }
+        if keyword not in split_dict.keys():
+            raise Exception(f'Keyword {keyword} does not exist.')
+        # Combine files
+        with open(output_fname, 'w') as out_file:
+            for core in range(ncores):
+                tmp_file = Path(path, keyword + "_" + str(core) + ".tsv")
+                with open(tmp_file) as infile:
+                    if core == 0:
+                        out_file.write(infile.read().rstrip("\n"))
+                    else:
+                        kw = split_dict[keyword]
+                        out_file.write(infile.read().split(kw)[1].rstrip("\n"))
+                self.log(f"File number {core} written")
```

Comment:
Are we sure about it? since these files are not big, I was thinking we could leave them in the caprieval folder to check for possible inconsistencies afterwards

Reply:
its not about the size its about the volume actually, haddock already generates way too many files and should try to minimize it the best we can.

Reply:
OK, agreed!